js/controllers/more-info.js

```diff
@@ -0,0 +1,21 @@
+import { Controller } from "/js/stimulus.js"
+
+export default class extends Controller {
+  static timeout = 15*60*1000; // 15 minutes
+  static targets = [ "modal" ]
+
+  connect() {
+  }
```

Comment:
in [the example in their docs](https://stimulus.hotwired.dev/handbook/working-with-external-resources#releasing-tracked-resources) they also threw in a `disconnect()` that cleans up the timer with `clearTimeout()`, which seems like a good move. i was looking this up yesterday and `clearTimeout()` with an invalid ID does absolutely nothing (just silently fails) so it seems safe to do always